org.jacoco.core/src/org/jacoco/core/instr/Instrumenter.java

```diff
@@ -114,7 +114,7 @@ protected String getCommonSuperClass(final String type1,
 				return instrument(new ClassReader(buffer));
 			}
 		} catch (final RuntimeException e) {
-			throw instrumentError(name, e);
+			throw instrumentError("class " + name, e);
```

Comment:
@Godin Absolutely! In the context of the agent and the class name it should be obvious that classes get instrumented.